rasa_nlu/extractors/duckling_extractor.py

```diff
@@ -78,12 +80,34 @@ def cache_key(cls, model_metadata):
 
         return cls.name + "-" + model_metadata.language
 
-    def process(self, text, entities):
+    def pipeline_init(self, language):
```

Comment:
This function needs to be removed (I don't think it is necessary anyways), it most likely is an artifact from resolving the merge conflicts with master. This function is also the reason the tests fail.